config-linux.md

```diff
@@ -637,6 +637,29 @@ The following parameters can be specified to set up seccomp:
     "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
 ```
 
+### <a name="configLinuxRdmaLimits" />RDMA resource limits
+
+**`rdma_limits`** (array of objects, OPTIONAL) limits resources for network interfaces which support RDMA (e.g. HCA handles).
+For more information, see the [the kernel RMDA controller documentation][cgroup-v1-rdma].
+
+Each entry has the following structure:
+
+* **`interface_name`** (string, REQUIRED) - A name of an interface.
+* **`hca_handle_limit`** (int64, REQUIRED) - Maximum number of HCA Handles (use `-1` to request the maximum available).
+* **`hca_object_limit`** (int64, REQUIRED) - Maximum number of HCA Objects (use `-1` to request the maximum available).
```

Comment:
From the kernel docs, I think `hca_handle_limit` and `hca_object_limit` should both be OPTIONAL, possibly with a requirement that at least one of them be set (this is the same suggestion I made for #942 [here][1]).

[1]: https://github.com/opencontainers/runtime-spec/pull/942#discussion_r155593193